switchapi/switch_pd_p4_name_mapping.h

```diff
@@ -156,6 +156,7 @@ extern "C" {
         "linux_networking_control.set_nexthop_id"
 
 #define LNW_ECMP_HASH_SIZE 65536
+#define LNW_MAX_MEMBERS 512
```

Comment:
Is this for nhop group members? If yes, can we rename the macro to reflect the same?